.vscode/launch.json

```diff
@@ -13,29 +13,34 @@
             "args": [
                 "--extensionDevelopmentPath=${workspaceRoot}"
             ],
-            "outFiles": [
-                "${workspaceRoot}/client/out/**/*.js"
-            ]
+            "outFiles": ["${workspaceRoot}/client/out/**/*.js"]
         },
         {
-            "type": "node",
-            "request": "attach",
-            "name": "Attach to Server",
-            "port": 6009,
+			"type": "node",
```

Comment:
we should probably normalize on tabs or spaces - editorconfig is usally a good tool for that

Reply:
do you already have a .editorconfig for typescript projects in general?

Reply:
Best ask in #typescript in Slack

Reply:
This file still has tabs ;)

Reply:
not anymore ;)  (accidentally removed the prettier file in a later commit)

Reply:
https://github.com/prisma/vscode/blob/i33/.vscode/launch.json still has tabs for me:
![image](https://user-images.githubusercontent.com/183673/80256977-2e5cf000-8680-11ea-9797-5d406d5e2fc8.png)
(Possible that this is only visible on GH with Refined Github extension installed)

Reply:
You were right, it still was'nt fixed, I had to change some paths in the prettier script, it only included the client and server, no files in the root folder.